src/sasjs-help/index.js

```diff
@@ -101,18 +101,15 @@ export async function printHelpText() {
       'context <command>'
     )} - performs operations on contexts.
         * ${chalk.cyanBright('create')} - creates new context.
-          1) Provide target name.
-          2) Provide path to context config file (eg. ./contextConfig.json).
+          example: sasjs create myTarget -source ../contextConfig.json
```

Comment:
myTarget is a misleading context name - in SASjs nomenclature, a target is a SAS server.  

which is a good point, we should add a target parameter, and default to the first target if a target is not found in the sasjsconfig file

Reply:
fixed